definitions/npm/antd_v3.x.x/flow_v0.25.x-/antd_v3.x.x.js

```diff
@@ -1,52 +1,64 @@
+import {Component, Node} from 'react';
```

Comment:
These should be moved inside module declaration: If they are on top of file they returns `any`

Reply:
ok, I have moved them now